content/api/commands/intercept.md

```diff
@@ -120,6 +145,12 @@ See [Request/Response Modification with `routeHandler`](#Request-Response-Modifi
 
 ## Examples
 
+<Alert type="info">
```

Comment:
Should there be a `### Spying` example?

Reply:
Yeah, but as we talked about, that's not in the scope of this PR.